### PR TITLE
[EDMTP- 248] Add accordion, add select options to branch

### PR DESCRIPTION
**Pull request by ldzmitrenka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,6 +216,27 @@ public class TestTabs {
...
 ```
+
+### @Accordion annotation
+
+To define array of `@Accordion` within `@Dialog` annotation. Then, to settle a field to a certain accordion you will need  to add `@PlaceOnAccordion` annotation to this particular field.  The values of `@PlaceOnAccordion` must correspond to the *title* value of the desired accordion. This is a somewhat more flexible technique which avoids creating nested classes and allows freely moving fields. You only need to ensure that tab title is specified everywhere in the very same format, no extra spaces, etc.
+```java
+@Dialog(
+    name = "test-component",
+    title = "test-component-dialog",
+    accordionTabs = {
```
Comment (lexastef) [inline README.md:227]: Isn't it a standalone component?

**Comment by liubou-masiuk** [comment]
Hi, @ldzmitrenka, a couple of concerns:
**1) (critical) Functionality.** 
- `variant`, `margins` and `multiple` don't seem to work at all. I don't see them anywhere in XML.
- properties mentioned above need to be specified for the whole accordion, not each panel.
- there is no ability to specify `active` or `disabled` properties which are mentioned in the spec

**2) Naming.** I suggest renaming `accordionTabs` to `accordions` or `accordionPanels` and `@PlaceOnAccordion` to `@PlaceInAccordion`
**3) Validation.** Сurrently, the user can specify both `tabs` and `accordionTabs`, and use both `@PlaceOnAccordion` and `@PlaceOnTab` annotations. However, only accordions are rendered in this case. Maybe it makes sense to add some kind of validation to warn the user?
**4) DependsOn.** there is no ability to conditionally hide accordion panels with dependsOn. Seems like we need something like `DependsOnPanel`. Probably this can be done in a separate ticket. @smiakchilo, @lexastef, what do you think?

File: aem-authoring-toolkit-api/src/main/java/com/exadel/aem/toolkit/api/annotations/container/PlaceOn.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+/**
+ * In multi-tab TouchUI Dialog setup, used to specify on which tab a dialog field is placed
+ */
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+@SuppressWarnings("unused")
+public @interface PlaceOn {
+    /**
+     * String equal to appropriate {@link Accordion#title()} value or {@link Tab #title()}
```
Comment (Alexey-Chernyakov) [inline aem-authoring-toolkit-api/src/main/java/com/exadel/aem/toolkit/api/annotations/container/PlaceOn.java:29]: redundant space

**Review by Alexey-Chernyakov — CHANGES_REQUESTED** [review]
I think we must do something with _com.exadel.aem.toolkit.core.handlers.container.AccordionHandler_ and _com.exadel.aem.toolkit.core.handlers.container.TabsHandler_.
Actually, they do the same work, but with different annotations.
We need to make a kind of AbstractContainerHandler.

**Pull request closed without merging**